integration-tests/verification-flow/bbs-plus-revocation.test.ts

```diff
@@ -8,8 +8,86 @@ import {createVerificationController} from '@docknetwork/wallet-sdk-core/src/ver
 import {verifyPresentation} from '@docknetwork/sdk/utils/vc/presentations';
 import axios from 'axios';
 import {ProofTemplateIds, createProofRequest} from '../helpers/certs-helpers';
+const testAPIURL = process.env.TESTING_API_URL || null;
+const testCredsURL = testAPIURL.replace('api-', 'creds-');
 
 const credential = {
+  '@context': [
```

Comment:
I just added a new file to export all the credential data for this test. I can't use the credential as json because we are using some creds from certs staging. ```${testCredsURL}/a68bd2dae567e146777655f2ec6bff9451c00ec48fd404731ae8822c2429ebc0```